src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpOnFilenameCheck.java

```diff
@@ -100,7 +100,9 @@
  * Type is {@code java.lang.String[]}.
  * Default value is {@code ""}.</li>
  * </ul>
- *
+ * <p>
+ * To configure the check:
+ * </p>
  * <p>
  * To configure the check to report file names that contain a space:
```

Comment:
`To configure the default check that report ...`

Reply:
Done